zilencer/models.py

```diff
@@ -63,10 +63,18 @@ class RemotePushDeviceToken(AbstractPushDeviceToken):
 
     server: RemoteZulipServer = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
     # The user id on the remote server for this device
-    user_id: int = models.BigIntegerField(db_index=True)
+    user_id: int = models.BigIntegerField(db_index=True, null=True)
```

Comment:
This `db_index` on `user_id` has literally never been used, because of the unique index which is a better prefix key.  It'd be nice to drop it while we're in here.